config.json

```diff
@@ -48,5 +48,18 @@
   },
   "concepts": [],
   "key_features": [],
-  "tags": []
-}
+  "tags": [
+    "paradigm/imperative",
+    "paradigm/procedural",
+    "typing/static",
+    "typing/weak",
+    "execution_mode/compiled",
+    "platform/android",
+    "platform/ios",
+    "platform/mac",
+    "platform/linux",
+    "runtime/standalone_executable",
+    "used_for/embedded_systems",
+    "used_for/mobile"
+  ]
```

Comment:
Sweet!

So for the JavaScript track, I created [this TAGS.md document](https://github.com/exercism/javascript/blob/main/TAGS.md) to explain to future readers _why_ we tagged with certain tags. This is absolutely optional, but might be nice for those less familiar with this language?